src/controllers/NotificationController.js

```diff
@@ -0,0 +1,15 @@
+import helpers from '../helpers';
+import services from '../services';
+
+const { notificationServices: { getUserNotification } } = services;
+
+
+const { responseMessage } = helpers;
+
+const getNotification = async (req, res) => {
+  const { id } = req.user;
+  const userNotificationObject = await getUserNotification(id);
```

Comment:
I feel you could use a `try/catch` block in this controller, in case an error occurs while awaiting `getUserNotification(id)`

Reply:
Pls attend to this review

Reply:
done